### DISCUSSION TITLE
Comments on a merged PR sometimes disappear

### DISCUSSION THREAD

**Discussion by noraa-junker** [community/community#10254] [Mobile]
https://user-images.githubusercontent.com/58633848/149989709-eedbd601-88a9-4c03-810d-bad4ea6e289c.mp4

As you can see in the video, if I open a PR notification in Github mobile the last comment disappers for no reason. After I reload it's there again, but disappears after a time.

Tjis only happens if the PR is merged

**Comment by brianlovin** [comment]
Hey @Aaron-Junker — that's definitely not great. I'm trying to reproduce this on my end, but so far haven't been able to. Perhaps it's an edge case with that particular PR. Are you able to reproduce this bug on any other notifications or PRs?

**Reply by noraa-junker** [reply]
@brianlovin I can reproduce tbis on any PR that got merged and which I access from the notifications.

**Reply by brianlovin** [reply]
Appreciate the context, I'll pass this along and see what we can find.

**Reply by noraa-junker** [reply]
@brianlovin Just wanted to inform you that this still happens. https://user-images.githubusercontent.com/58633848/159264282-51e24088-13f0-433d-b4a9-0b7674120be3.mp4

**Reply by brianlovin** [reply]
@Aaron-Junker hm I can't seem to view that video. Could you re-upload, or email me? I'm brianlovin at GitHub dot com 🙏

**Comment by stevepopovich** [comment]
@Aaron-Junker are you still seeing this? I can't reproduce this anymore and we may have fixed this with some other improvements.

**Reply by noraa-junker** [reply]
Yes it still happens

**Reply by stevepopovich** [reply]
Hmm. I am on the exact the PR in the video on the latest Android version and don't see the issue. Are you on the most recent version of the app. Android right?

**Reply by noraa-junker** [reply]
> Hmm. I am on the exact the PR in the video on the latest Android version and don't see the issue. Are you on the most recent version of the app. Android right?

Yes. Android. It happens on any closed PR

**Reply by stevepopovich** [reply]
What version of the app?

**Reply by noraa-junker** [reply]
@stevepopovich  v1.79.0-beta (10083)

**Reply by stevepopovich** [reply]
Great. Thanks for the info and quick responses! I'll be honest, we are having lots of trouble reproducing so it may take some more time to resolve. Thanks for the patience!

**Reply by stevepopovich** [reply]
@Aaron-Junker Does this happen on push notifications? Or when you manually search for the PR?

**Reply by stevepopovich** [reply]
Sorry to bother @Aaron-Junker, checking in again here as we are still tracking this. Do this happen via notification or manually searching for a PR? Did you try a reinstall?

**Reply by noraa-junker** [reply]
> Sorry to bother @Aaron-Junker, checking in again here as we are still tracking this. Do this happen via notification or manually searching for a PR? Did you try a reinstall? 

It only happens when I open it from the notifications.

Yes. My phone even did a full reset and it still happens.

https://user-images.githubusercontent.com/58633848/195587233-c1071985-4888-4727-94fe-99f3b4fb1ee6.mp4

**Reply by stevepopovich** [reply]
Ah, thank you so much!
